chatbot/conversation_start.php

```diff
@@ -17,7 +17,7 @@
 $thisFile = __FILE__;
 
 /** @noinspection PhpIncludeInspection */
-require_once("../config/global_config.php");
+require_once(_CONF_PATH_ . 'global_config.php');
```

Comment:
this really doesn't make any sense, as the constant `_CONF_PATH_` is defined in the config file that you're calling, so first off  the config file will never be found, and second, if it were (by defining that constant before the `require_once` call), there would end up being defined twice, and that's not allowed in PHP.